src/modules/campaign-monitor/services/campaign-monitor.service.ts

```diff
@@ -114,17 +241,21 @@ export class CampaignMonitorService {
 				return;
 			}
 
-			url = `${process.env.CAMPAIGN_MONITOR_API_ENDPOINT as string}/${
-				templateIds[emailInfo.templateId]
+			const url = `${
+				process.env.CAMPAIGN_MONITOR_TRANSACTIONAL_SEND_MAIL_API_VERSION as string
+			}/${process.env.CAMPAIGN_MONITOR_TRANSACTIONAL_SEND_MAIL_API_ENDPOINT as string}/${
+				templateIds[emailInfo.template]
 			}/send`;
 
-			const data: any = {
-				Data: emailInfo.data,
-				ConsentToTrack: 'unchanged',
-			};
+			const data: any = emailInfo.data;
+
+			if (isArray(emailInfo.data.to) && emailInfo.data.to.length > 1) {
+				const emailInfoDataTo = emailInfo.data.to;
+
+				data.To = [head(emailInfoDataTo)];
 
-			if (isArray(emailInfo.data.to)) {
-				data.BCC = emailInfo.data.to;
+				emailInfoDataTo.shift();
+				data.BCC = emailInfoDataTo;
 			} else {
 				data.To = [emailInfo.data.to];
 			}
```

Comment:
This will not work correctly when emailInfo.data.to is an array with only one element. better:
```typescript
			if (isArray(emailInfo.data.to)) {
				const emailInfoDataTo = emailInfo.data.to;

				data.To = [head(emailInfoDataTo)];

				emailInfoDataTo.shift();
				data.BCC = emailInfoDataTo;
			} else {
				data.To = [emailInfo.data.to];
			}
```